text/0000-issue-templates.md

```diff
@@ -0,0 +1,110 @@
+
+## Implementation
+
+1. Update `.github` under the `paketo-buildpacks` Github org with the agreed upon templates.
+2. Create `.github` under the `paketo-buildpacks` Github org.
+3. Add the agreed upon templates.
+
+## Prior Art
+
+There is an existing template (prior art), which implies that we are generally sure that having templates is a good idea.
+
+## Unresolved Questions and Bikeshedding
+
+1. Do we need additional templates beyond the one existing and two newly proposed templates?
+
+2. What is in each template? Including templates below.
+
+### Submit a Bug
+
+```markdown
+<!--- Provide a general summary of the issue in the Title above -->
+
+## Expected Behavior
+<!--- Tell us what should happen -->
+
+## Current Behavior
+<!--- Tell us what happens instead of the expected behavior -->
+
+## Possible Solution
+<!--- Not obligatory, but suggest an acceptable fix for the bug -->
+
+## Steps to Reproduce
+<!--- Provide a link to an reproduction/test case, or an unambiguous set of steps to reproduce this bug. Include notes about the software versions you're using, the environment in which you're running, code snippets to reproduce, log output, screenshots, etc.. -->
+1.
+
+## Motivations
+<!--- How has this issue affected you? What are you trying to accomplish? What is the impact? Providing context helps us come up with a solution that is most useful in the real world. -->
+```
+
+### Submit a Feature Request
+
+```markdown
+<!--- Provide a general summary of the request in the Title above -->
+
+## Describe the Enhancement
+<!--- Explain the change you'd like to see. Include information about how the buildpack works now and how you envision it to work after this change. -->
+
+## Possible Solution
+<!--- Not obligatory, but suggest an acceptable fix for the bug -->
+
+## Motivation
+<!--- Why do you want to see this change? What are you trying to accomplish? What is the impact? Providing context helps us come up with a solution that is most useful in the real world.  -->
+```
+
+### Report a security vulnerability
+
+```markdown
+<!--- Provide a general summary of the request in the Title above -->
+
+## Prerequisites
+
+* [ ] Has this security vulnerability been publicly disclosed already? **IF NO, [GO HERE](https://github.com/paketo-buildpacks/community/security/policy)**. Please do not ever disclose a security vulnerability for the first time in a public forum.
+
+* [ ] Is this a vulnerability in a dependency installed by the buildpack? **IF YES THEN STOP**, you need to open an issue with the upstream project following their security policy. Paketo Buildpacks will pick up the fix when upstream has made it available.
+
+* [ ] Is this a vulnerability detected by a scanner against a buildpack build/run image, stack, or buildpack generated image? **IF YES**, then [please read this first](https://paketo.io/docs/concepts/stacks/#when-are-paketo-stacks-updated). If it does not address your concern, please continue.
```

Comment:
I like this list of checks and actions - moves all the relevant information to one place that's close at hand for the issue filer.